app/models/server/models/LivechatAgentActivity.js

```diff
@@ -4,9 +4,9 @@ export class LivechatAgentActivity extends Base {
 	constructor(...args) {
 		super(...args);
 
-		this.tryEnsureIndex({ date: 1 });
 		this.tryEnsureIndex({ agentId: 1, date: 1 }, { unique: true });
-		this.tryEnsureIndex({ agentId: 1 });
+		this.tryEnsureIndex({ lastStoppedAt: 1 });
```

Comment:
I found only with `lastStoppedAt: { $exists: false }`, if this is really the query using `lastStoppedAt` this index will not help

Reply:
Yup, that's the only one right now. I'm planning to change some queries around the feature and this index will come handy 🙈 so that's why it's here now